syne_tune/experiments/launchers/utils.py

```diff
@@ -13,7 +13,7 @@
 from pathlib import Path
 from typing import Optional
 
-from syne_tune.experiments.results_utils import sync_from_s3_command
+from syne_tune.experiments.visualization.results_utils import sync_from_s3_command
```

Comment:
Should `sync_from_s3_command` be under visualization?

Reply:
Moved to `launchers/utils`